libraries/joomla/document/document.php

```
@@ -397,14 +403,7 @@ public function getMetaData($name, $httpEquiv = false)
 		}
 		else
 		{
-			if ($httpEquiv == true)
-			{
-				$result = @$this->_metaTags['http-equiv'][$name];
-			}
-			else
-			{
-				$result = @$this->_metaTags['standard'][$name];
-			}
+			$result = @$this->_metaTags[$attribute][$name];
```

@andrepereiradasilva Why are we suppressing errors here? Can't we just check if the array key exists? It would be more elegant.